### PR TITLE
Nerfs silo scaling rating on Nuclear War

### PR DESCRIPTION
**Pull request by lKiseki**
## About The Pull Request
Per title.

## Why It's Good For The Game
Throughout recent rounds, xenos have proven to be _very_ numerous, and they even had several larvae to boot. Marines are unable to keep up with the sheer amount of xenos, and short of xenos making considerable mistakes, marines are unable to keep up.

This is the solution I propose: reduce the silo scaling and therefore reduce the overall number of xenos, given that I do not think it is particularly related to xeno balance or whatever, moreso just how many xenos there are in any given round.

## Changelog
:cl: Lewdcifer
balance: Nuclear War silo scaling reduced from 2 > 1.5 (this means less xenos).
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
Que chulo

**Comment by ClosetedSkeletonGH** [comment]
Just got out of a low pop round with like, 33 players total (no clue how many marines, maybe like 14-16) and like 9 xenos and we had 4-5 T3's, it was ridiculous

**Comment by maikilangiolo** [comment]
So the issue is that the silo gen is overtuned? I was wondering how the hell there were that many xenos. A high pop round had constant 15+ xenos around and they took more than a dozen casualties within an hour

**Comment by Strayed-Raven** [comment]
This is why xenos always have pop for king despite wiping by being greedy

**Comment by Naaanii** [comment]
Doesn't this only nerf the respawn rate rather than the total number of Xenos/Larvae?

**Comment by Aporhtonoma** [comment]
queue xenos going back to distress balance levels while still having the extra marine win condition because :marines refuse to try to do that extra win condition and refuse to not suicide inside the  maze 5 minite into the round

**Comment by lKiseki** [comment]
> Doesn't this only nerf the respawn rate rather than the total number of Xenos/Larvae?

image

TL;DR the larva spawn rate is multiplied by the silo scaling. So, no, this also affects the spawn rates.

**Comment by DeltaFire15** [comment]
> Throughout recent rounds, xenos have proven to be _very_ numerous, and they even had several larvae to boot. Marines are unable to keep up with the sheer amount of xenos, and short of xenos making considerable mistakes, marines are unable to keep up.

At which point does nuclear just become "distress but marines have a bonus win condition", considering this is the like what, third, silo rate nerf.

If marines cannot keep up with xeno numbers enough to push caves as is tradition, maybe they should have been doing the thing the mode is named after before benos can ramp up.

**Comment by lKiseki** [comment]
> > Throughout recent rounds, xenos have proven to be _very_ numerous, and they even had several larvae to boot. Marines are unable to keep up with the sheer amount of xenos, and short of xenos making considerable mistakes, marines are unable to keep up.
> 
> At which point does nuclear just become "distress but marines have a bonus win condition", considering this is the like what, third, silo rate nerf.
> 
> If marines cannot keep up with xeno numbers enough to push caves as is tradition, maybe they should have been doing the thing the mode is named after before benos can ramp up.

You either nerf their numbers or enhance marine ability to clear mazes and other structures (normal walls, for example). Otherwise, you get the current state of balance where xenos do not really suffer from losses since they can just invalidate them if you give their silo(s) a few minutes.

**Comment by Aporhtonoma** [comment]
beno spawns have gone from x3 to x2.5 to x2 and now to x1.5 its kinda like you antipower creep em at this point.the reason xenos are numerous is because marines make bad decisions and get drained by the xenos repeadetly(especially robot players which have no personal disadvantage to being drained) instead of nerfing xenos even more incentivise marines to not dying to another cave push instead.One change is a cell damage equilevent to robots so they actually play a bit more carefully (reposted and edited cause i said a bad word  :<)

**Comment by Sebbe9123** [comment]
> > > Throughout recent rounds, xenos have proven to be _very_ numerous, and they even had several larvae to boot. Marines are unable to keep up with the sheer amount of xenos, and short of xenos making considerable mistakes, marines are unable to keep up.
> > 
> > 
> > At which point does nuclear just become "distress but marines have a bonus win condition", considering this is the like what, third, silo rate nerf.
> > If marines cannot keep up with xeno numbers enough to push caves as is tradition, maybe they should have been doing the thing the mode is named after before benos can ramp up.
> 
> You either nerf their numbers or enhance marine ability to clear mazes and other structures (normal walls, for example). Otherwise, you get the current state of balance where xenos do not really suffer from losses since they can just invalidate them if you give their silo(s) a few minutes, and marines can't really fathom a proper push since it's all mazed or there's wacky chokepoints.

Are there many maps where you actually have to go into caves to get to a disk in nukewar? Afaik, most maps can be mazecleared with mortars, which is how it's done right now from my experience.

**Comment by RipGrayson** [comment]
Might as well throw in my opinion here. Currently xenos are too oppressive, either they need scaling reduced or mazes nerfed in some fashion (either easier to destroy or less of them). Frequently marines can't go more than 40 tiles from their LZ before a wipe, let alone visit and hold all three disk generators.

**Comment by Pariah919** [comment]
Jokes aside, xenomorph scale way better in the early game in nuclear war because they can deal with the attirtion marines give them due to the amount of respawns they get, which lets them snowball harder before marine power gear gets into the game. I think the spawn rates are too high considering the inherent disadvantage marines are as attackers but another option could be to give rewards for accomplishing disks to marines like random assortment of gear in supply drops or something which drives people to want to go get the disks.

But this is probably better, so eh.

**Comment by AlexCrimson** [comment]
This is probably needed. However i think it is also worth noting that this will punish Xeno mistakes even harder. When you have Xeno players that keep feeding(it happens, not a big deal), the 2nd silo is what allows Xenos to put up any kind of resistance to a Marine win.

This change is essentially making poor Marine coordination less punishing, whilst hurting Xenos ability to make a comeback.

Again though, i will say that building more silos is incredibly effective. If Xenos snowball enough to get a 2nd silo early, then Marines will have a very hard time keeping up. Maybe a different solution is needed? Rather than nerf the scaling, lock out 2nd silos unless there is less than a certain # of Xenos alive, or just make it time gated? No 2nd silo before x minutes after the first one?

**Comment by Barnet2** [comment]
Xeno deaths are, at the moment, unpunished. This is due to maturity and evolution towers, which Xenos can get a lot more of due to how abundant psy points are nowadays. I wish for the long larva queue again because perhaps Xenos would value their life more, I like the direction this change is going.

**Comment by Strayed-Raven** [comment]
> Xeno deaths are, at the moment, unpunished. This is due to maturity and evolution towers, which Xenos can get a lot more of due to how abundant psy points are nowadays. I wish for the long larva queue again because perhaps Xenos would value their life more, I like the direction this change is going.

this. A xeno died earlier today as elder runner and came back within no time as mature runner with very little downtime.

**Comment by AlexCrimson** [comment]
>I mean same for marines, isn't it? Why should only marines get punished for a certain % of their players always feeding or doing FF or breaking cades etc, and xenos not?

Marines can jump back into a fight easier than Xenos. Literally just defib them. Who cares about some FF then? Whereas Xenos FF is usually shuffling someone which results in their death, and Xenos can only come back by taking up a larva, which is not really an infinite resource.

And the response to excessive Marine FF was to add Aim Mode, no?

**Comment by Pariah919** [comment]
> >I mean same for marines, isn't it? Why should only marines get punished for a certain % of their players always feeding or doing FF or breaking cades etc, and xenos not?
> 
> Marines can jump back into a fight easier than Xenos. Literally just defib them. Who cares about some FF then? Whereas Xenos FF is usually shuffling someone which results in their death, and Xenos can only come back by taking up a larva, which is not really an infinite resource.
> 
> And the response to excessive Marine FF was to add Aim Mode, no?

Body blocking response was shuffle.
Excessive FF response was the damage multiplier, not aimmode. Aimmode was an attempt to remove SGs, didn't work but we kept both at the very least..

**Comment by Losenis** [comment]
>If marines cannot keep up with xeno numbers enough to push caves as is tradition, maybe they should have been doing the thing the mode is named after before benos can ramp up.

If marines don't have the numbers to do a single coordinated push towards silo, they won't have the numbers to go all around the map to each of the three disks and then go to nuke. Usually when they have done disks, it was either on the back of a silopush getting every xeno distracted, or after a xeno wipe that'd have just ended earlier with a silopush. Disks as they currently are amount to hardly more than an antidelay tool to force the round to end.

It also doesn't help that some disks or nuke are right next to the most popular silo placements. On Big Red it's usually lambda or around ETA that the silo is placed, so while green and red disks are on the way there, either you grab blue and silopush to ETA silo instead of going all the way to nuke, or you silopush straight to lambda instead of grabbing the disks first and going there anyway due to nuke placement.

**Comment by Deroh-Ra** [comment]
Bro just add distress back into the vote list, whole point of war is to be a high pop game mode.

**Comment by maikilangiolo** [comment]
> > If marines cannot keep up with xeno numbers enough to push caves as is tradition, maybe they should have been doing the thing the mode is named after before benos can ramp up.
> 
> If marines don't have the numbers to do a single coordinated push towards silo, they won't have the numbers to go all around the map to each of the three disks and then go to nuke. Usually when they have done disks, it was either on the back of a silopush getting every xeno distracted, or after a xeno wipe that'd have just ended earlier with a silopush. Disks as they currently are amount to hardly more than an antidelay tool to force the round to end.
> 
> It also doesn't help that some disks or nuke are right next to the most popular silo placements. On Big Red it's usually lambda or around ETA that the silo is placed, so while green and red disks are on the way there, either you grab blue and silopush to ETA silo instead of going all the way to nuke, or you silopush straight to lambda instead of grabbing the disks first and going there anyway due to nuke placement.

Xenos truly underestimate what it takes to do disks. If marines don't push silo and just do disks, they will get surrounded ane attritioned just like if they sat still in fob.

**Comment by Arturlang** [comment]
Honestly, if we want to make xeno lives more valuable, then give xenos in critical as much critical health as they have positive health, so that any xeno in critical can't just be one-shotted by a marine, and gives xenos around them actual time to save them, but that's probably a good suggestion for a different PR

**Comment by Lumipharon** [comment]
Nothing has changed in particular that I can see that would warrant this. We've just been following the normal winrate curve of copeage.

**Comment by isotope436** [comment]
after seeing a round with 26 xenos 6 burrowed and multipled dead xenos at only 50 alive marines yes it still needs to be nerfed
